js-intermediate.js

```diff
@@ -62,11 +104,35 @@ var middleLetters1 = "hello"
 var middleLetters2 = "rhinoceros"
 // Expected output: “oc”
 
+const middleLet = (string) => {
+	let letterArr = string.split("")
+	if (letterArr.length % 2 !== 0) {
+		return onlyMiddle = letterArr.slice(letterArr.length / 2, letterArr.length / 2 + 1)
+	}else {
+		return onlyMiddle = letterArr.slice(letterArr.length / 2 - 1, letterArr.length / 2 + 1)
+	}
+}
+console.log(middleLet(middleLetters1));
+console.log(middleLet(middleLetters2));
 
 
 
 // 7. Write a program to get the area of a sphere using object classes. Create three spheres with different radi as test cases. Area of a sphere =  4πr^2 (four pi r squared)
 
+var sphere1 = {
+	radius: 4
+}
+var sphere2 = {
+	radius: 15
+}
+var sphere3 = {
+	radius: 3
+}
+
+const findSphereArea = (mySphere) => {
+	let sphereArea = null;
+	return sphereArea = 4 * Math.PI * (Math.pow(mySphere.radius,2))
+}
 
 
```

Comment:
You are getting the correct output but the goal is to create a class that handles the logic and sphere objects that are an instance of the class.